administrate/config/locales/administrate.de.yml

```diff
@@ -0,0 +1,23 @@
+---
+de:
+  administrate:
+    actions:
+      confirm: Sind sie sicher?
+      destroy: Löschen
+      edit: Editieren
+      show: Anzeigen
+    controller:
+      create:
+        success: "%{resource} wurde erfolgreich erstellt."
+      destroy:
+        success: "%{resource} wurde erfolgreich gelöscht."
+      update:
+        success: "%{resource} wurde erfolgreich aktualisiert."
+    fields:
+      has_many:
+        more: %{count} von %{total_count}
```

Comment:
It looks like the CI build is failing because you can't start a translation with a `%`. Can you either wrap this in quotes, or add a word in the front? Thanks!